src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -2492,4 +2522,68 @@ namespace winrt::Microsoft::Terminal::Control::implementation
     {
         _playWarningBell->Run();
     }
+
+    Windows::UI::Xaml::DependencyProperty VisualBellLight::m_isTargetProperty =
+        Windows::UI::Xaml::DependencyProperty::RegisterAttached(
+            L"IsTarget",
+            winrt::xaml_typename<bool>(),
+            winrt::xaml_typename<winrt::Microsoft::Terminal::Control::VisualBellLight>(),
+            Windows::UI::Xaml::PropertyMetadata{ winrt::box_value(false), Windows::UI::Xaml::PropertyChangedCallback{ &VisualBellLight::OnIsTargetChanged } });
+
+    void VisualBellLight::OnConnected(Windows::UI::Xaml::UIElement const& /* newElement */)
+    {
+        if (!CompositionLight())
+        {
+            // OnConnected is called when the first target UIElement is shown on the screen. This enables delaying composition object creation until it's actually necessary.
+            auto spotLight2{ Windows::UI::Xaml::Window::Current().Compositor().CreateAmbientLight() };
+            spotLight2.Color(Windows::UI::Colors::WhiteSmoke());
+            spotLight2.Intensity(static_cast<float>(1.5));
+            CompositionLight(spotLight2);

```

Comment:
```suggestion
            auto spotLight{ Windows::UI::Xaml::Window::Current().Compositor().CreateAmbientLight() };
            spotLight.Color(Windows::UI::Colors::WhiteSmoke());
            spotLight.Intensity(static_cast<float>(1.5));
            CompositionLight(spotLight);
```
What happened to `spotLight1`? haha

Reply:
whoops, was trying different lights and forgot to rename haha